CrossPlatform/BasePage.cs

```diff
@@ -16,18 +16,16 @@ public abstract class BasePage
 
         protected abstract Trait Trait { get; }
 
-        protected Func<AppQuery, AppQuery> CurrentTrait { get; private set; }
-
         protected BasePage()
         {
             app = AppInitializer.App;
 
             OnAndroid = app.GetType() == typeof(AndroidApp);
             OniOS = app.GetType() == typeof(iOSApp);
 
-            CurrentTrait = OnAndroid ? Trait.AndroidTrait : Trait.iOSTrait;
+            Trait.Platform = OnAndroid ? Platform.Android : Platform.iOS;
 
-            if (CurrentTrait == null)
+            if (Trait.CurrentTrait == null)
```

Comment:
Picking nits here, but what about calling it `Trait.Current` rather than `Trait.CurrentTrait`? Just seems a bit redundant. Same for `Trait.Android` and `Trait.iOS`